src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Onramps style guide
+contentType: page
+template: basicDoc
+topics:
+  - onramps
+---
+
+import apmOnrampsIssuesExample from 'images/apm_screenshot-crop_onramps-issues-example.webp'
+
+import apmActiveEmptyState from 'images/apm_screenshot-crop_active-empty-state.webp'
+
+Onramps content is information that appears our UI when a user clicks the **?** icon. This content is meant to be truncated and contextualized to where the user is and what they need to know to perform a task.
+
+## How should I name the section titles?
+
+When applicable, use an FAQ format where the headers are in question format.
+
+For example, here's the Issues tile with headers formatted into questions.
```

Comment:
Should issues be bolded? I'm not actually sure myself so I figure it can't hurt to ask!